README.md

```diff
@@ -0,0 +1,7 @@
+# Beginning Web Development 
+
+## Important Links
+* [Mockups by Level](https://www.figma.com/design/8GS4B4rOXS7OKaY5H4g2br/BWD-Design-Project?node-id=0-1&t=uDAFUoXuth5RSgD7-0)
```

Comment:
A few other people from the team helped with the mockups. You should probably say something like "Mockups by the webdev team".